app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,147 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DemoHttpHandler implements HttpHandler {
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private List<Task> tasks = new ArrayList<>();
+    private Long id = 1L;
+    private int failStatusCode = 404;
+    private int successStatusCode = 200;
```

Comment:
한결 낫습니다. 그런데 이 값은 변경하지 않을 상수로 사용하게 될 테니 `static final`로 선언해보시는 것을 권합니다. 기왕이면 Http Status의 정식 명칭을 사용해 다음과 같이 할 수도 있겠죠.

```suggestion
    private int NOT_FOUND = 404;
    private int OK = 200;
```

[RFC 2616]( https://tools.ietf.org/html/rfc2616#section-10 )의 HttpStatus 항목도 읽어보시길 권합니다. RFC 2616은 HTTP를 만든 팀 버너스 리가 쓴 문서입니다. 1997년에 RFC2068이 작성됐고, RFC2616은 1999년에 개정된 것입니다.